src/descriptors/power_system_structs.json

```diff
@@ -1903,7 +1903,8 @@
         {
           "null_value": "false",
           "name": "status",
-          "data_type": "Bool"
+          "data_type": "Bool",
```

Comment:
This should be defined properly by the user

Reply:
Removed